src/com/amazon/ionschema/internal/constraint/TimestampOffset.kt

```diff
@@ -43,6 +43,7 @@ internal class TimestampOffset(
      * exactly with values returned by IonTimestamp.localOffset.
      */
     private val validOffsets: Set<Int?>
```

Comment:
Now that there's a `not::` option, we don't know whether this `Set` is storing `validOffsets` or `invalidOffsets`. I suggest changing it to just `offsets`.